<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStickerCreate</class>
 <widget class="QDialog" name="DialogStickerCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="LBL_DLG_STICKER_ADD_TITLE">Add Sticker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lh3.qrc">
    <normaloff>:/lh3/Resources/operators_stickers.png</normaloff>:/lh3/Resources/operators_stickers.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string comment="LBL_DLG_STICKER_ADD_TITLE_L">Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leTitle"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string comment="LBL_DLG_STICKER_ADD_MESSAGE">Message</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="ptMessage"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_CREATE">Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_CANCEL">Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lh3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DialogStickerCreate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogStickerCreate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
